src/main/java/tars/UI.java

```diff
@@ -0,0 +1,147 @@
+package tars;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Handles the user interface for the TARS application, providing methods
+ * to read user input and display responses.
+ *
+ * <p>The UI class is responsible for interacting with the user, displaying
+ * messages, and receiving commands through the console. It provides various
+ * methods to print different types of messages, such as task updates, errors,
+ * and greetings.
+ */
+public class UI {
```

Comment:
Perhaps the class name could be changed to Ui to comply with the coding standards which states that abbreviations and acronyms should not be uppercase when used as a part of a name.